README.md

```diff
@@ -1,8 +1,27 @@
-Confluent's Apache Kafka client for Golang
-==========================================
+Confluent's Golang Client for Apache Kafka<sup>TM</sup>
```

Comment:
I'm a bit confused about the discussion because both versions read fine to me...